### ISSUE TITLE
Shortfin LLM example gives gibberish tokens

### ISSUE THREAD

**Issue by renxida** [nod-ai/shark-ai#287]
# Symptom

<details>
<summary>
Shark tank generates sensible tokens
</summary>

```
python -m sharktank.examples.paged_llm_v1 --gguf-file=/tmp/sharktank/llama/open-llama-3b-v2-f16.gguf "1 2 3 4 5" --tokenizer-config-json /tmp/sharktank/llama/tokenizer_config.json
```

Output:
1 2 3 4 5 6 7 8 9 10 11
</details>

<details>
<summary>
Shortfin does not. Generates repetitive gibberish instead.
</summary>

code ran: https://gist.github.com/renxida/29f7f4d29bb7a94d7049457faa0bc9a9

output:
```
Generate endpoint status code: 200
Generated text:
</nd---------------------
```
</details>

# Things I'm trying

## Is it a tokenizer problem?

Probably not:

https://gist.github.com/renxida/74c50fb620d1f2ba35db088bc0d18005

```
(env1) (base) xidaren2@Shark22:~/SHARK-Platform/convos$ python tokenizer_test.py 
Testing Sharktank Tokenizer:
You set `add_prefix_space`. The tokenizer needs to be converted from the slow tokenizers
Encoded (token IDs): [1, 8479, 29522, 1173, 29578, 660, 325, 260, 1248, 296, 268, 8206, 4498, 3152, 601, 29520]
Lengths: [16]
Decoded: ['<s> Hello, world! This is a test of the tokenizer implementations.']

Testing Shortfin Tokenizer:
Encoded (token IDs): [1, 8479, 29522, 1173, 29578, 660, 325, 260, 1248, 296, 268, 8206, 4498, 3152, 601, 29520]
Decoded: ['Hello, world! This is a test of the tokenizer implementations.']
```

**Comment by renxida** [comment]
# Do the logits look normal?

Not at all!
1st output token has nan logits
every output token after that has zero logits for EVERYTHING!

Logs:

https://gist.github.com/renxida/156d151189f5ec831f3eb9bb16434281
